packages/scripts/src/TypeChatJsonTranslateFunc.ts

```diff
@@ -48,13 +49,12 @@ export function makeTypeChatJsonTranslateFunc<SchemaType extends object>({
     apiKey,
     `${baseUrl}openai/deployments/${deployment}/chat/completions?api-version=${version}`
   );
-
-  // LLM function
-  const translator = createJsonTranslator<SchemaType>(
-    model,
+  const validator = createTypeScriptJsonValidator<SchemaType>(
```

Comment:
New TypeChat update? I don't see a version bump or anything.

Reply:
yeah

Reply:
I don't see a version bump in the diff - do we need to update `scripts/package.json`?

Reply:
maybe? i'll set the version in package.json to whatever is most recent

Reply:
version in package.json is the latest. i'm confused what's going on, but will keep as is since it works and i dont think this usage is important enough for us to worry too much about